ui/src/containers/Header/Header.css

```diff
@@ -10,9 +10,50 @@
 }
 
 .header-sigin-button {
-  margin-bottom: 2em;
+  margin-top: 2em;
 }
 
 .header-sigin-button__icon {
   float: left;
 }
+
+.header-login__modal {
+  border-radius: 0.5em;
+  width: 21em;
+}
+
+.header-login__modal svg {
+  margin-right: -1em;
+  vertical-align: 0em !important;
+}
+
+.header-sigin-button a {
+  outline: none;
+}
+
+.header-sigin-button:hover {
+  color: white;
+  background-color: #0066cc;
+}
+
+.header-login__modal > header > h1 {
+  text-align: center;
+  margin-left: 1em;
+}
+
+.header-sigin-button a:hover {
+  color: white;
+}
+
+.header-sigin-button a:hover svg {
+  color: white;
+}
+
+.header-login__modal h6 {
+  margin-bottom: -0.5em !important;
+  text-align: center;
+}
+
+.pf-c-backdrop {
```

Comment:
I think we should not modify the root component

Reply:
May be a component which we are adding inside that we can add a className and then modify it

Reply:
yes, but in hub we are using patternfly modal in two components and if we want to blur the background (as require same css fields) then we need at root level css otherwise we need to define same css in both component css file where patternfly modal have used

Reply:
May be then where we are using the component there we can give inline styles